src/main/msc/emfat_file.c

```diff
@@ -316,7 +316,7 @@ static int emfat_find_log(emfat_entry_t *entry, int maxCount, int flashfsUsedSpa
     int buffOffset;
     int hdrOffset;
     int fileNumber = 0;
-    uint8_t buffer[HDR_BUF_SIZE];
+    static uint8_t buffer[HDR_BUF_SIZE];
```

Comment:
Betaflight convention is to have static variable declarations at the start of functions.

Reply:
Done